### PR TITLE
Issue 4115: Change logging level to Debug for system tests

### LINKED ISSUE #4115: Change the logging level to DEBUG in system tests
**Problem description**
Currently, TRACE logging is enabled on client side for system tests.

**Problem location**
System tests

**Suggestions for an improvement**
Change logging level to TRACE

### PR DESCRIPTION
**Pull request by anishakj**
Signed-off-by: anishakj <anisha.kj@dell.com>

**Change log description**  
Change the client side logging of system tests to `DEBUG` level. This is for both K8 and docker swarm tests.

**Purpose of the change**  
Fixes #4115

**What the code does**  
Logging level is changed

**How to verify it**  
System tests should not log at TRACE level.

### CODE REVIEW

File: test/system/src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@ You may obtain a copy of the License at
         </encoder>
     </appender>
 
-    <root level="TRACE">
+    <root level="DEBUG">
```
Comment (shrids) [inline test/system/src/test/resources/logback-test.xml]: The jenkins job or command line can specify the log level  using `-Dlog.level=DEBUG`

The log level is not passed to the test pods and we should ensure the system properties on the test pod are updated correctly.

https://github.com/pravega/pravega/blob/38b1b001f896c39748cc2cd084a834f6ed28c804/test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/K8SequentialExecutor.java#L123-L124

https://github.com/pravega/pravega/blob/38b1b001f896c39748cc2cd084a834f6ed28c804/test/system/src/main/java/io/pravega/test/system/framework/DockerBasedTestExecutor.java#L173-L176

Comment (anishakj) [reply test/system/src/test/resources/logback-test.xml]: Addressed the comments.

**Comment by RaulGracia** [comment]
@anishakj have you verified that the log level in the tests is correct?

**Comment by anishakj** [comment]
> @anishakj have you verified that the log level in the tests is correct?

Yes, I have verified logs are coming fine

**Comment by anishakj** [comment]
> +1 , The changes look good.
> 
> * The build failure was due to an unrelated unit test failure (ContainerKeyIndexTests > testRecoveryTimeout), have retriggered the build.
> * Please indicate in the PR description/title that the changes are specific to Kubernetes based system tests.

The changes are for both K8 and docker swarm system tests

**Pull request merged**